src/tutorial/runtime/introduction.rst

```diff
@@ -291,65 +337,165 @@ Where Do All The Files Go?
 
          ~/cylc-run/<workflow-name>/log/job/<cycle-point>/<task-name>/<job-submission-num>/
 
-      .. image:: ../img/cylc-gui-view-log.png
-         :align: center
-         :scale: 75%
+      .. TODO REPLACE THIS IF APPROPRIATE
 
+         .. image:: ../img/cylc-gui-view-log.png
+            :align: center
+            :scale: 75%
 
-Running A Workflow
-------------------
+
+Installing A Workflow
+---------------------
 
 .. ifnotslides::
 
-   It is a good idea to check a workflow for errors before running it.
-   Cylc provides a command which automatically checks for any obvious
-   configuration issues called ``cylc validate``, run via:
+   .. seealso::
 
-.. code-block:: sub
+      :ref:`The full guide to Cylc install <Installing-workflows>`.
+
+   To allow you to separate the development and running of workflows
+   Cylc provides a :term:`cylc install <install>` command.
+
+
+From ``cylc-src/my_workflow``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: bash
 
-   cylc validate <path/to/workflow>
+   cylc install my_workflow
 
 .. ifnotslides::
 
-   Here ``<path/to/workflow>`` is the path to the workflow's location within the
-   filesystem (so if we create a workflow in ``~/cylc-run/foo`` we would put
-   ``~/cylc-run/foo/flow.cylc``).
+   will install your workflow in ``~/cylc-run/my_workflow/runN``.
 
-   Next we can run the workflow using the ``cylc play`` command.
+From a workflow developed in an arbitrary location
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. code-block:: sub
 
-   cylc play <name>
+   cylc install -C /some/location/my_other_workflow
 
 .. ifnotslides::
 
-   The ``name`` is the name of the :term:`run directory` (i.e. ``<name>``
-   would be ``foo`` in the above example).
+   will install your workflow in ``~/cylc-run/my_other_workflow/runN``.
 
-.. note::
+   .. note::
+
+      You can still develop workflows in ``~/cylc-run``, but this is not
+      recommended because development work may
+      change the behaviour of the workflow as it is running.
+
+Validating A Workflow
+---------------------
+
+.. ifnotslides::
 
-   In this tutorial we are writing our workflows in the ``cylc-run`` directory.
+   It is a good idea to check a workflow definition for errors before running
+   it. Cylc provides a command which automatically checks the validity of
+   workflow configurations - ``cylc validate``:
+
+.. code-block:: console
+
+   $ cylc validate <path/to/workflow>
+   $ cylc validate <workflow_id>  # For workflow in `~/cylc-run/`
+
+.. ifnotslides::
+
+   Here ``<path/to/workflow>`` is the path to the workflow's location within the
+   root source directory (e.g. ``~/cylc-src/``).
```

Comment:
There's no point installing then validating as `cylc play` now does all the validate stuff.

Perhaps introduce validate before install.

```
The ``cylc validate`` command can help to spot errors in the workflow before you install it.
```

There's an issue with the `<path/to/workflow>` substitution caused by a change with the UID branch. Relative references must now be explicit e.g:

```
.
./
./foo
./foo/bar
```

As opposed to implicit e.g:

```
foo
foo/bar
```

So `path/to/workflow` will now fail.

Reply:
I've moved the location of the introduction to before `cylc install` and removed the reference to absolute paths. I think that it's a good idea to inculcate the idea of validation before install.